bowtie/templates/report.html.j2

```diff
@@ -410,13 +413,100 @@
+                  data-bs-custom-class="custom-tooltip"
+                  data-bs-title="Copy to clipboard">
+                  {# Clipboard SVG Icon  #}
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-clipboard" viewBox="0 0 16 16">
+                      <path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z"/>
+                      <path d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z"/>
+                    </svg>
+                  </button>
+                </div>
+          </div>
+        <div id="instance-info" class="card-body"></div>
+        </div>
+      </div>
+    </div>
+  {# Schema Display end  #}
   {% endfor %}
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
   <script>
     const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="tooltip"]')
     const tooltipList = [...tooltipTriggerList].map(tooltipTriggerEl => new bootstrap.Tooltip(tooltipTriggerEl))
   </script>
-
+  {# Scripts for Schema Display start  #}
+  <script>
+      const displayBody = document.querySelector("#display-body");
+      const codes = document.querySelector("#schema-code");
+      function displayCode(code, targetId) {
+        const targetDisplay = document.getElementById(targetId);
+        displayBody.classList.remove("d-none");
+        displayBody.classList.add("d-block");
+        targetDisplay.innerHTML = `<pre>${code}</pre>`;
+      }
+      function schemaDisplay(parentBodyId,code,targetId) {
+        displayBody.classList.remove("d-none");
+        displayBody.classList.add("d-block");
+        displayCode(code,targetId);
+        document.getElementById('instance-info').innerHTML = '';
+        const isDisplayed = displayBody.parentElement === document.getElementById(parentBodyId);
+        if (isDisplayed) {
+          displayBody.parentElement.removeChild(displayBody);
+        } else {
+          const accordionBody = document.querySelector(`#${parentBodyId}`);
+          accordionBody.insertBefore(displayBody, accordionBody.firstChild);
+        }
+      }
+      {# Copy Schema from display  #}
+      function copyToClipboard(targetTextToCopy, btnId){
+        const textToCopy = document.getElementById(targetTextToCopy).innerText;
+        const tempElement = document.createElement("textarea");
+        tempElement.value = textToCopy;
+        document.body.appendChild(tempElement);
+        tempElement.select();
+        tempElement.setSelectionRange(0, 99999);
+        document.execCommand("copy");
```

Comment:
I think this is the old API for doing this -- on not-super-old browsers I think you can just use [the clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API).